.vscode/extensions.json

```
@@ -0,0 +1,8 @@
+{
+  "recommendations": [
+    "dbaeumer.vscode-eslint",
+    "esbenp.prettier-vscode",
+    "bradlc.vscode-tailwindcss",
+    "svelte.svelte-vscode"
```

Pretty TypeScript Errors might be a nice one to add here as well

```suggestion
    "svelte.svelte-vscode",
    "yoavbls.pretty-ts-errors"
```